py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -148,6 +154,11 @@ def compile(
         hardware_compatible (bool): Build the TensorRT engines compatible with GPU architectures other than that of the GPU on which the engine was built (currently works for NVIDIA Ampere and newer)
         timing_cache_path (str): Path to the timing cache if it exists (or) where it will be saved after compilation
         lazy_engine_init (bool): Defer setting up engines until the compilation of all engines is complete. Can allow larger models with multiple graph breaks to compile but can lead to oversubscription of GPU memory at runtime.
+        cache_built_engines (bool): Whether to save the compiled TRT engines to storage
```

Comment:
Minor: Consider renaming this to `save_engine_cache`

Reply:
It was `save_engine_cache`. after discussion with Naren, we deem `cache_built_engines` might be better.